application/routes/audio.py

```
@@ -35,22 +35,27 @@
-
+import logging
+logger = logging.getLogger(__name__)
 
 @api.route('/audio', methods=['POST'])
 def start_audio():
+    #TODO: Request recieved with body
+    logger.debug("Audio: Received request to start audio with data: {}".format(request.json))
     session_id = request.json.get('session_id')
     audio, reason = create_connection(session_id, ConnectionType.AUDIO)
     if reason != '':
+        logger.warning("Audio: Failed to create audio connection: {}".format(reason))
```

Ruff

Use f-string instead of `format` call

Convert to f-string

(UP032)